### PR TITLE
Add RichMenu related model class.

### PR DESCRIPTION
**Pull request by kazuki-ma**
RichMenu related Model class and serialize/deserialize test for that.

Relates #142 
See https://developers.line.me/en/docs/messaging-api/reference/#rich-menu

Need confirmation: Using ParameterNames module to read parameter names from constructor.

### CODE REVIEW

**Comment by kazuki-ma** [comment]
Adding models for RichMenu. Review please @line/bot-team .

File: line-bot-api-client/src/main/java/com/linecorp/bot/client/LineMessagingServiceBuilder.java [hunk 1]
```diff
@@ -201,12 +202,13 @@ public LineMessagingService build() {
...
-        final ObjectMapper objectMapper = new ObjectMapper();
-        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
-
-        // Register JSR-310(java.time.temporal.*) module and read number as millsec.
-        objectMapper.registerModule(new JavaTimeModule())
-                    .configure(DeserializationFeature.READ_DATE_TIMESTAMPS_AS_NANOSECONDS, false);
+        final ObjectMapper objectMapper = new ObjectMapper()
+                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+                // Register ParameterNamesModule to read parameter name from lombok generated constructor.
+                .registerModule(new ParameterNamesModule())
```
Comment (tokuhirom) [inline line-bot-api-client/src/main/java/com/linecorp/bot/client/LineMessagingServiceBuilder.java]: I guess ParameterNamesModule is no longer required.

Comment (kazuki-ma) [reply line-bot-api-client/src/main/java/com/linecorp/bot/client/LineMessagingServiceBuilder.java]: ParameterNamesModule is used to read mapping names from constructor name.
(Suppressing `@JsonParameter` annotation is possible by this)

I think supressing writing fieldname twice is good.

Can I use ParameterNamesModule?

**Comment by kazuki-ma** [comment]
Oh... I miss understaing.
If we use `ParameterNamesModule`, `@JsonCreator` annotation is needed.

> if there are multiple visible constructors and there is no default constructor, @JsonCreator is required to select constructor for deserialization

**Comment by kazuki-ma** [comment]
Unn... without `@JsonCreator` annotation, looks like can't deserialize object with single field or generics field...

Review current code please~.

File: line-bot-model/src/main/java/com/linecorp/bot/model/richmenu/RichMenuCommomProperties.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package com.linecorp.bot.model.richmenu;
+
+import java.util.List;
+
+public interface RichMenuCommomProperties {
```
Comment (be-hase) [inline line-bot-model/src/main/java/com/linecorp/bot/model/richmenu/RichMenuCommomProperties.java]: What's `Commom` ?
Do you mean `Common` ?

**Pull request merged**